cluster/gce/config-test.sh

```diff
@@ -611,10 +611,22 @@ export CLOUD_PROVIDER_FLAG="${CLOUD_PROVIDER_FLAG:-external}"
 export ENABLE_AUTH_PROVIDER_GCP="${ENABLE_AUTH_PROVIDER_GCP:-false}"
 
 # External cloud provider requires ENABLE_AUTH_PROVIDER_GCP and feature flags
-# DisableKubeletCloudCredentialProviders and DisableCloudProviders
+# DisableKubeletCloudCredentialProviders and DisableCloudProviders should be set to true
 if [[ "${CLOUD_PROVIDER_FLAG:-}" == "external" ]]; then
+  if [[ "${FEATURE_GATES:-}" == *"DisableKubeletCloudCredentialProviders=false"*  ]]; then
```

Comment:
i think it would be good for debugging to add the `echo "before FEATURE_GATES: ${FEATURE_GATES:-}"` line here as well, it looks like the failure in the `pull-kubernetes-e2e-gce` test might be from here since we don't see the "before FEATURE_GATES" output.